### ISSUE TITLE
Pleae keep https://github.com/scribusproject updated on your progress

### ISSUE THREAD

**Issue by luzpaz** [berteh/ScribusGenerator#2]
Looks promising!

**Comment by luzpaz** [comment]
CC @aoloe @JLuc @FirasH @andreas-vox

**Comment by berteh** [comment]
Thanks for the interest!
What do you mean by "keep us updated"?
 
For the time being I have no plans to improve this project as it work satisfactorily... but if you have particular ideas/wishes kindly let me know!

B.

**Comment by luzpaz** [comment]
We're from the Scribus Project https://github.com/scribusproject 
It's the official github repo for Scribus.

**Comment by luzpaz** [comment]
@berteh would you mind making a screencast showing how this works ?

**Comment by berteh** [comment]
I don't mind at all, good idead. Will try to do it in the next days.
B.

**Comment by berteh** [comment]
took me quite some time, sorry. screencast for Scribus Generator is now available. hope this helps.
https://www.youtube.com/watch?v=kMsRn38TOiQ

@aoloe @JLuc @FirasH @andreas-vox 

Any feedback is welcome. I'm especially interested in getting the sla->pdf time down... As you can see from https://www.youtube.com/watch?feature=player_detailpage&v=kMsRn38TOiQ#t=245 it takes about 30 sec for one simple sla to load and get exported to pdf (this time does not increase much with complexity, but it's too much to generate 50+ files at once).
One option could be (maybe) to concatenate all generated sla documents in a single sla... and generate a single pdf from all of it. Does that seem feasible or is likely to blow up?

**Comment by luzpaz** [comment]
another scribus-dev question

**Comment by aoloe** [comment]
concatenating the SLAs might work if they all have the same settings/characteristics (fonts, colors, color management, targetted pdf version, ...)

getting scribus to perform better would prpobably be the better solution :-)

what you can do, is using a profiler to try to understand where scribus is spending all that time and checking if those actions are really needed (in that form / in headless mode) or if they might be cached.

**Comment by JLuc** [comment]
warning : concatenating sla becomes a mess with styles conflicts. Dont do it unless styles are all strictly identical  or have all strictly different names.

**Comment by bjarnesvanberg** [comment]
@JLuc I don't think that will be a problem for this project as the source is always a single .sla file from which the content is extracted and repeated.

**Comment by berteh** [comment]
@JLuc and @bjarnesvanberg  indeed, it's not a problem in the particular case of ScribusGenerator, as the "header" (styles, meta-info, master-info,...) is exactly similar for all generated pages, so we can easily concatenate the content (just updating the page count and objects position)... and it works great ;)
